clustermesh-apiserver/clustermesh/root.go

```diff
@@ -360,8 +360,9 @@ func startServer(
 	config := cmtypes.CiliumClusterConfig{
 		ID: cinfo.ID,
 		Capabilities: cmtypes.CiliumClusterConfigCapabilities{
-			SyncedCanaries:       true,
-			MaxConnectedClusters: cinfo.MaxConnectedClusters,
+			SyncedCanaries:        true,
+			MaxConnectedClusters:  cinfo.MaxConnectedClusters,
+			ServiceExportsSupport: true,
```

Comment:
If it's for client reporting I believe we should somehow have a way to know if the feature is not supported, disabled (and supported) or enabled. As the sync to the kvstore mark as synced even if the feature is disabled we only need to do some special handling for a remote cluster in case the feature is not supported.